code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -40,10 +40,10 @@
 /*||||||||||||||/----------\||||||||||||||*\
 ||||||||||||||||[GAS DATUMS]||||||||||||||||
 ||||||||||||||||\__________/||||||||||||||||
-||||These should never be instantiated. ||||
-||||They exist only to make it easier   ||||
-||||to add a new gas. They are accessed ||||
-||||only by meta_gas_list().            ||||
+|||| These should never be instantiated. ||||
+|||| They exist only to make it easier   ||||
+|||| to add a new gas. They are accessed ||||
+|||| only by meta_gas_list().            ||||
 \*||||||||||||||||||||||||||||||||||||||||*/
```

Comment:
🗞️ undocced change

Reply:
ctrl-h hit this and I thought it was funny for some reason (that reason being I'm dumb)

Reply:
this is alright honestly you can leave it